docs/auth/README.adoc

```diff
@@ -13,3 +13,118 @@ dependencies {
 
 == Usage
 
+To use the Auth SDK you'll first need to import the Core module. See link:../core/README.adoc[the Core documentation].
+
+=== Authenticating
+
+To start the authentication invoke the `AuthService#performAuthRequest` method.
```

Comment:
I think this needs to be updated to reflect the right name of the method. It should be `login`?